images/linux/scripts/installers/edge.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+################################################################################
+##  File:  edge.sh
+##  Desc:  Installs Edge
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
```

Comment:
Do we really need this helper here?

Reply:
No, this does not affect the process in any way, I removed helper